src/python/turicreate/test/test_object_detector.py

```diff
@@ -329,6 +328,7 @@ def test_export_coreml(self):
 
     @unittest.skipIf(_mac_ver() < (10, 14),
         "Non-maximum suppression is only supported on MacOS 10.14+.")
+
```

Comment:
Why are we adding a blank line here? Does it affect the scope of the annotation?

Reply:
I'll delete it